lib/bforest/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+authors = ["The Cranelift Project Developers"]
+name = "cranelift-bforest"
+version = "0.18.1"
+description = "A forest of B+-trees"
+license = "Apache-2.0"
```

Comment:
Cranelift code uses "Apache-2.0 WITH LLVM-exception".

Reply:
Please also add a copy of the LICENSE file in lib/bforest (so that people pulling it in as a dependency get a copy).